twitter/config/const.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+return [
+    'MAX_LENGTH' => 20,
+    'NAME_MIN_LENGTH' => 2,
+    'NAME_MAX_LENGTH' => 20,
```

Comment:
```suggestion
        'NAME' => [
        'MIN' => 2,
        'MAX' => 20,
    ],
```
`config('const.NAME.MIN')`こんな書き方もできるよ

Reply:
こちらの書き方の方が可読性が上がっていいですね！
教えていただきありがとうございます！